Sources/Build/BuildDescription/SwiftModuleBuildDescription.swift

```diff
@@ -541,14 +541,21 @@ public final class SwiftModuleBuildDescription {
             args += ["-emit-module-interface-path", self.parseableModuleInterfaceOutputPath.pathString]
         }
 
-        if self.buildParameters.prepareForIndexing {
+        switch self.buildParameters.prepareForIndexing {
+        case .off:
+            break
+        case .on:
             if !args.contains("-enable-testing") {
                 // enable-testing needs the non-exportable-decls
                 args += ["-Xfrontend", "-experimental-skip-non-exportable-decls"]
             }
             args += [
                 "-Xfrontend", "-experimental-skip-all-function-bodies",
                 "-Xfrontend", "-experimental-lazy-typecheck",
+            ]
+            fallthrough
+        case .noSkipping:
```

Comment:
`-experimental-skip-non-exportable-decls` and `-experimental-lazy-typecheck` are the flags we'd want to turn off. `-experimental-skip-all-function-bodies` and `-experimental-allow-module-with-compiler-errors` are well-used at this point, so are pretty unlikely to cause issues.

Reply:
Good call 👍🏽